### DISCUSSION TITLE
Add "Delay between actions" to tasks

### DISCUSSION THREAD

**Discussion by MarkimusDev** [gameserverapp/Platform#292] [Suggestions]
When an entire cluster, or multiple servers are selected for a task to be executed on, please add a "Delay between actions" dropdown for us to set a delay that will be respected between each server the action is sent to.

So if I have a cluster with 12 servers, and I select the entire cluster for the action to be sent to, then set a delay of 10 seconds (or 10 minutes, etc), it will send the task to server 1, wait, server 2, wait, and so on.

**Comment by mvaessen** [comment]
Hi, this is now available for `Scheduled` task!
Available options: no delay, 20 second delay, 40 second delay
